vnext/ReactUWP/Views/ReactControl.cpp

```diff
@@ -144,6 +152,9 @@ void ReactControl::DetachRoot() noexcept
     m_touchEventHandler->RemoveTouchHandlers();
   }
 
+  if (!m_previewKeyboardEventHandlerOnRoot)
```

Comment:
Inconsistent formatting. Earlier in the change (107) you wrap this in braces, here you don't